bundles/org.openhab.binding.folderwatcher/README.md

```diff
@@ -33,14 +33,27 @@ The `localfolder` thing has the following configuration options:
 | pollIntervalLocal | Polling interval, s | Interval for polling folder changes                                                                       | yes      | 60            |
 | listRecursiveLocal | List Sub Folders | Allow listing of sub folders                                                                                | yes      | No            |
 
+The `s3bucket` thing has the following configuration options:
+
+| Parameter   | Name         | Description                                                                                                            | Required | Default value |
+|-------------|--------------|------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| S3BucketName    | S3 bucket name | Name of the S3 bucket to be watched                                                                              | yes      | n/a           |
```

Comment:
For all these parameters, please use camel casing. I would suggest s3BucketName , s3Path, s3PollIntervall, awsKey, awsSecret, awsRegion, anonymouseConnection. (maybe that last one is a bit long)